lib/vfscore/automount.c

```diff
@@ -346,10 +350,86 @@ extern const char vfscore_einitrd_start[];
 extern const char vfscore_einitrd_end;
 #endif /* CONFIG_LIBVFSCORE_AUTOMOUNT_EINITRD */
 
+#define LIBVFSCORE_EXTRACTOPT_MOUNT_RAMFS		(0x1 << 0)
+#define LIBVFSCORE_EXTRACTOPT_MOUNT_IFNOMOUNT		(0x1 << 1)
+#define LIBVFSCORE_EXTRACTOPT_FORMAT_AUTO		(0x1 << 2)
+
+static inline int vfscore_extract_parse_opts(const char *opts)
+{
+	char strbuf[64];
+	size_t cpylen;
+	const char *arg, *iter;
+	size_t arglen;
+	ssize_t match;
+	int optflags = 0x0;
+
+	for (iter = opts; iter != NULL; ) {
```

Comment:
Can we just go with s/iter != NULL/iter/

Reply:
of course, fine.